src/main/java/de/cyklon/reflection/function/Sorter.java

```diff
@@ -0,0 +1,31 @@
+package de.cyklon.reflection.function;
+
+import de.cyklon.reflection.types.Loadable;
+import de.cyklon.reflection.types.Modifiable;
+import de.cyklon.reflection.types.Nameable;
+
+import java.util.Comparator;
+
+public interface Sorter<T> extends Comparator<T> {
+
+	static <T> Sorter<T> random() {
+		return (o1, o2) -> Math.round((float) (Math.random()*3-2));
```

Comment:
Why?

Reply:
At the moment there is no real benefit, but in some cases you have to shuffle a list or something.

Reply:
I still don't see the purpose. Why should this library provide a random sorter? This library will not need this for sure and if someone needs this they should implement it them self. This is out of scope for this lib.